TpmTestingPkg/TpmReplayPeiDxe/InputChannel/TpmReplayFfsFileDxe.c

```diff
@@ -0,0 +1,37 @@
+/** @file

```

Comment:
I'd like to keep them for now. The UefiVariableDxe logic actually does work with the common code and the FFS logic might be used in the near future. Since the common code has the hooks, I'd like to keep this in case someone else or I need to fill it in, it will be obvious where to do so.

Reply:
I ended up removing it.